test/truefi2/TrueFiPool2.test.ts

```diff
@@ -160,6 +158,57 @@ describe('TrueFiPool2', () => {
     })
   })
 
+  describe('liquidValue', () => {
+    const includeFee = (amount: BigNumber) => amount.mul(10000).div(9975)
```

Comment:
Thanks for moving this inside the smallest scope it's needed in!

Could you rename `includeFee()` to be more explicit? It took me multiple readthroughs of your tests to realize this is linked with the pool's 0.25% joining fee. Something like `include025PercentJoiningFee()`, perhaps?